File: OPENAPI-REST-API/swagger-client/csharp-dotnet2/src/main/CsharpDotNet2/myadmin-client-csharp-dotnet2/Model/VpsTrafficHistorySectionResponse.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.Serialization;
using Newtonsoft.Json;

namespace IO.Swagger.Model {

  /// <summary>
  /// VPS Traffic History Hour and Day Sections
  /// </summary>
  [DataContract]
  public class VpsTrafficHistorySectionResponse {
    /// <summary>
    /// Gets or Sets Data
    /// </summary>
    [DataMember(Name="data", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "data")]
    public List<VpsTrafficHistorySectionDataResponse> Data { get; set; }

    /// <summary>
    /// Gets or Sets Times
    /// </summary>
    [DataMember(Name="times", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "times")]
    public List<DateTime?> Times { get; set; }


    /// <summary>
    /// Get the string presentation of the object
    /// </summary>
    /// <returns>String presentation of the object</returns>
    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class VpsTrafficHistorySectionResponse {\n");
      sb.Append("  Data: ").Append(Data).Append("\n");
      sb.Append("  Times: ").Append(Times).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }

    /// <summary>
    /// Get the JSON string presentation of the object
    /// </summary>
    /// <returns>JSON string presentation of the object</returns>
    public string ToJson() {
      return JsonConvert.SerializeObject(this, Formatting.Indented);
    }

}
}

File: OPENAPI-REST-API/swagger-client/dart/lib/model/vps_traffic_history_section_response.dart
part of swagger.api;

class VpsTrafficHistorySectionResponse {
  
  List<VpsTrafficHistorySectionDataResponse> data = [];

  List<DateTime> times = [];

  VpsTrafficHistorySectionResponse();

  @override
  String toString() {
    return 'VpsTrafficHistorySectionResponse[data=$data, times=$times, ]';
  }

  VpsTrafficHistorySectionResponse.fromJson(Map<String, dynamic> json) {
    if (json == null) return;
    data = VpsTrafficHistorySectionDataResponse.listFromJson(json['data']);
    times = DateTime.listFromJson(json['times']);
  }

  Map<String, dynamic> toJson() {
    return {
      'data': data,
      'times': times
     };
  }

  static List<VpsTrafficHistorySectionResponse> listFromJson(List<dynamic> json) {
    return json == null ? new List<VpsTrafficHistorySectionResponse>() : json.map((value) => new VpsTrafficHistorySectionResponse.fromJson(value)).toList();
  }

  static Map<String, VpsTrafficHistorySectionResponse> mapFromJson(Map<String, Map<String, dynamic>> json) {
    var map = new Map<String, VpsTrafficHistorySectionResponse>();
    if (json != null && json.length > 0) {
      json.forEach((String key, Map<String, dynamic> value) => map[key] = new VpsTrafficHistorySectionResponse.fromJson(value));
    }
    return map;
  }
}
